src/FlexVotingDelegatable.sol

```diff
@@ -0,0 +1,84 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import {SafeCast} from "@openzeppelin/contracts/utils/math/SafeCast.sol";
+import {Context} from "@openzeppelin/contracts/utils/Context.sol";
+import {Checkpoints} from "@openzeppelin/contracts/utils/structs/Checkpoints.sol";
+
+import {FlexVotingBase} from "src/FlexVotingBase.sol";
+
+/// @notice This is an abstract contract designed to make it easy to build
+/// clients for governance systems that inherit from GovernorCountingFractional,
+/// a.k.a. Flexible Voting governors.
+///
+/// This contract extends FlexVotingBase, adding the ability to subdelegate one's
+/// internal voting weight. It is meant to be inherited from in conjunction with
+/// FlexVotingClient. Doing so makes the following usecase possible:
+///   - user A deposits 100 governance tokens in a FlexVotingClient
+///   - user B deposits 50 governance tokens into the same client
+///   - user A delegates voting weight to user B
+///   - a proposal is created in the Governor contract
+///   - user B expresses a voting preference P on the proposal to the client
+///   - the client casts its votes on the proposal to the Governor contract
+///   - user B's voting weight is combined with user A's voting weight so that
+///     150 tokens are effectively cast with voting preference P on behalf of
+///     users A and B.
+abstract contract FlexVotingDelegatable is Context, FlexVotingBase {
```

Comment:
Naming bikeshed: some googling seems to indicate "Delegatable" is not a real word but "Delegable" is. I'm not saying that definitely means we should change it, but I think we should consider it.

Reply:
I'm fine with the change. I think they're equally clear and all else being equal I'd rather we use real words 😄 

Done in 3e16491